src/display/editor/annotation_editor_layer.js

```diff
@@ -679,16 +513,15 @@ class AnnotationEditorLayer {
     }
 
     for (const editor of this.#editors.values()) {
-      this.removePointerInTextLayer(editor);
+      this.#accessibilityManager?.removePointerInTextLayer(editor.contentDiv);
       editor.isAttachedToDOM = false;
       editor.div.remove();
       editor.parent = null;
     }
-    this.#textNodes.clear();
     this.div = null;
     this.#editors.clear();
-    this.#waitingEditors.clear();
     this.#uiManager.removeLayer(this);
+    this.#accessibilityManager.disable();
```

Comment:
Given that the `accessibilityManager` is also used for the "regular" annotationLayer, which may not be removed at the same time as this editor, is it actually correct to have this `disable`-call here?